api/forms.py

```diff
@@ -85,3 +87,30 @@ def listing_submit():
 
     response = {'status': 200}
     return response
+
+
+# Route for submitting password reset:
+@forms.route('/reset-password-submit', methods=['POST'])
+def reset_pass_submit():
+    """Reset password submission route.
+    This function handles the submission of a password reset.
+    """
+    data = request.json
+    response = dict()
+    user = ListingsModel.query.get(data["username"])
```

Comment:
1.  Stick to single quotes for consistency.
2. When I put my username, I get an ID error in the SQL query.

Reply:
> 1. Stick to single quotes for consistency.
> 2. When I put my username, I get an ID error in the SQL query.

I took a look, its using my username as an ID in the query.